src/api/components/CommentList/CommentList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Comment } from '../Comment/Comment';
+
+import './CommentList.scss';
+
+export const CommentList = ({ commentList }) => (
+  <ul className="CommentList">
+    {commentList.map(comment => (
+      <li key={comment.id} className="CommentList__item">
+        <Comment {...comment} />
+      </li>
+    ))}
+  </ul>
+);
+
+CommentList.propTypes = {
+  commentList: PropTypes.arrayOf(
+    PropTypes.shape({}),
```

Comment:
It would be better to describe `comment.id` here, because you use it in this component